### DISCUSSION TITLE
API Method > "points" in config.json file

### DISCUSSION THREAD

**Discussion by ciscosian** [jkctech/Telegram-Trilateration#9] [General]
On API Method > Getting started > Edit config.json to your preference and place in your API ID and hash step you have kind of points as below: 

"points": [
		[52.9435548, 4.721217],
		[52.9294912, 4.7842015],
		[52.9607238, 4.7702266]
	]

from where you get this datas?

I edited api_id and api_hash, then running nearby.py and opening .json file on webview.html. But it shows me incorrect locations.

**Comment by jkctech** [comment]
You can get coordinates from Google Maps.

But, Telegram has FINALLY listened and lowered the accuracy of the "People Nearby" function. So this software will not function in it's current state anymore anyway.
